src/staff-handbook/leave/holiday.md

```diff
@@ -3,30 +3,17 @@ title: Holiday
 related_order: 1
 last_reviewed_at: ""
 ---
-We offer 25 days of annual leave plus bank holidays (England & Wales entitlement) each calendar year. If you are working part-time or on a fixed-term contract, the amount of holiday you are entitled to will be pro-rated.
+We offer 25 days of holiday plus bank holidays (England entitlement) each calendar year. If you are working part-time or on a fixed-term contract, the amount of holiday you are entitled to will be pro-rated.
 
 The holiday year at dxw runs between 1st January and 31st December, and you are able to carry over up to a maximum of 5 days of unused holiday to the following year. This carry over will be pro-rated for anyone on a part-time or non standard working pattern.
 
 ## Booking holiday
 
-1. Discuss the dates with your team and delivery lead
-
-   This gives them a chance to manage any impact it might have on the wider team
-   and the project. Colleagues in GovPress should do this via the `#dxw-govpress-holiday-plans`
-   Slack channel.
-2. Request the holiday through BreatheHR
-
-   We use BreatheHR to track who's off and when, so we can plan for it. Once you’ve requested a holiday, your line manager will review your request.
-3. Add your holiday to your Google calendar as 'Out of office'
-
-   If someone is trying to find out where you are, or when you're available, the
-   first thing they'll check is your calendar.
-
-   When you are on a client project, you may also need to add your holiday to a
-   shared or team calendar. Please check this with your delivery lead.
-4. Set your Slack status to 'On holiday'
-
-   Do this when you finish on your last day before your holiday, and set it to clear when you come back.
+1. Discuss the dates with your line manager, team and, if you’re in a billable role , your Delivery Lead.This gives them a chance to manage any impact it might have on the wider team and the project.
```

Comment:
```suggestion
1. Discuss the dates with your line manager, team and, if you’re in a billable role, your Delivery Lead. This gives them a chance to manage any impact it might have on the wider team and the project.
```
Just a couple of small space changes